### DISCUSSION TITLE
Providing an Endpoint to Check if the Connector is Working as Expected

### DISCUSSION THREAD

**Discussion by efiege** [eclipse-edc/Connector#2415] [Ideas]
For instance:

- Ensuring Data-Planes are accessible for the Control-Plane
- A DAT can be retrieved in centralized settings
- ...

**Comment by jimmarino** [comment]
Please review the Observability and Readiness infrastructure. 

Data Plane readiness/availability should not be handled by control plane runtimes. This can be handled by other deployment infrastructure.

DAT token retrieval and verification should not be part of the core runtime infrastructure since it is an optional identity provider.
